docs/DeveloperGuide.md

```diff
@@ -377,15 +376,15 @@ Use case ends.
 
 **MSS**
 
```

Comment:
Might be better to indicate that the list is the filtered / currently displayed list of members. (As opposed to full list of members)